### PR TITLE
Updated NEWS.md, cran-comments, re-run revdep

### PR DESCRIPTION
**Pull request by rsh52**
# Description
This PR is solely meant as a draft for `NEWS.md`, `cran-comments.md`, and to updated outputs of `revdep::revdep_check()` (no errors found). I modelled these after those on [`REDCapR`](https://github.com/OuhscBbmc/REDCapR/blob/main/cran-comments.md)'s page.

### PR Checklist
Before submitting this PR, please check and verify below that the submission meets the below criteria:

- [NA] New/revised functions have associated tests
- [NA] New/revised functions that update downstream outputs have associated static testing files (`.RDS`) updated under `inst/testdata/create_test_data.R`
- [NA] New tests that make API calls use `httptest::with_mock_api` and any new mocks were added to `tests/testthat/fixtures/create_httptest_mocks.R`
- [NA] New/revised functions use appropriate naming conventions
- [NA] New/revised functions don't repeat code
- [NA] Code changes are less than **250** lines total
- [NA] Issues linked to the PR using [GitHub's list of keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] The appropriate reviewer is assigned to the PR
- [x] The appropriate developers are assigned to the PR

# Code Review
This section to be used by the reviewer and developers during Code Review after PR submission

### Code Review Checklist

- [ ] I checked that new files follow naming conventions and are in the right place
- [ ] I checked that documentation is complete, clear, and without typos
- [ ] I added/edited comments to explain "why" not "how"
- [ ] I checked that all new variable and function names follow naming conventions
- [ ] I checked that new tests have been written for key business logic and/or bugs that this PR fixes
- [ ] I checked that new tests address important edge cases

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -8,6 +8,12 @@
 
 ### New Features
 
+* Function name changes:
+  * `read_redcap_tidy` is now `read_redcap`
```
Comment (skadauke) [inline NEWS.md:12]: Please always include `()` at the end of function names in docs, e.g. `read_redcap()` not `read_redcap`.

File: NEWS.md [hunk 2]
```diff
@@ -16,7 +22,7 @@
   * `data_size`: Size of the redcap_data tibble in memory
   * `data_na_pct`: The percentage of missing data in the corresponding redcap_data
 * New `make_labelled()` function allowing for descriptive labelling application to variables via the [`labelled`](https://larmarange.github.io/labelled/) package
-* New arguments available for `read_redcap_tidy` capabilities:
+* New arguments available for `read_redcap` capabilities:
```
Comment (skadauke) [inline NEWS.md:25]: "New arguments for `read_redcap`:`

File: NEWS.md [hunk 3]
```diff
@@ -16,7 +22,7 @@
...
   * `forms`: Allows for `forms` specification during API calls instead of defaulting to full database downloads
```
Comment (skadauke) [inline NEWS.md:26]: Allows importing specific instruments instead of the whole project

File: NEWS.md [hunk 4]
```diff
@@ -16,7 +22,7 @@
...
   * `export_survey_fields`: Allows for export of `[instrument_name]_timestamp` and `redcap_survey_identifier` fields related to survey-enabled REDCap databases (`TRUE` by default)
```
Comment (skadauke) [inline NEWS.md:27]: Change `[instrument_name]_timestamp` to `redcap_survey_timestamp`

File: revdep/README.md [hunk 5]
```diff
@@ -1,29 +1,27 @@
...
 
 |package      |old   |new   |Δ  |
 |:------------|:-----|:-----|:--|
-|REDCapTidieR |0.1.2 |0.1.2 |   |
-|crayon       |NA    |1.5.2 |*  |
-|jsonlite     |NA    |1.8.2 |*  |
-|readr        |NA    |2.1.3 |*  |
-|vctrs        |NA    |0.4.2 |*  |
-|vroom        |NA    |1.6.0 |*  |
+|REDCapTidieR |0.1.3 |0.1.3 |   |
```
Comment (skadauke) [inline revdep/README.md:21]: This says "new: 0.1.3". I guess we need to rerun this after bumping the version?

File: cran-comments.md [hunk 6]
```diff
@@ -1,27 +1,46 @@
 # Description
 
-This submission features a bug patch and a few minor documentation changes.
-
 Thank you for taking the time to review this submission and please reach out to either Stephan Kadauke or Richard Hanna for any questions, comments, or concerns.
 
-## R CMD check results:
+This submission includes new features, optimizations, and patches for a few identified bugs.
```
Comment (skadauke) [inline cran-comments.md:5]: ~and patches for a few identified bugs~ and bug fixes.

File: cran-comments.md [hunk 7]
```diff
@@ -1,27 +1,46 @@
...
+  * `extract_table` is now `extract_tibble`
+  * `extract_tables` is now `extract_tibbles`
+  * `bind_table` is now `bind_tibbles`
+    * All old functions are still supported and deliver a deprecation warning
+* The `read_redcap` output now includes additional helpful metadata
+* New `make_labelled()` function allowing for descriptive labelling application to variables via the [`labelled`](https://larmarange.github.io/labelled/) package
+* New arguments available for `read_redcap` capabilities:
+  * `forms`
+  * `export_survey_fields`
+
```
Comment (skadauke) [inline cran-comments.md:20]: Should we list the optimizations and bug fixes here?

File: cran-comments.md [hunk 8]
```diff
@@ -1,27 +1,46 @@
...
-- A NOTE is generated in R-hub Windows (Server 2022, R-devel 64-bit), a similar issue appears to arise in the `REDCapR` package and appears to be related to the R-hub test environment.
+### New Features
 
-```
-* checking for detritus in the temp directory ... NOTE
-Found the following files/directories:
-  'lastMiKTeXException'
-```
+* Function name changes:
+  * `read_redcap_tidy` is now `read_redcap`
```
Comment (skadauke) [inline cran-comments.md:10]: Please add `()` to function names

File: cran-comments.md [hunk 9]
```diff
@@ -1,27 +1,46 @@
...
+  * `extract_table` is now `extract_tibble`
+  * `extract_tables` is now `extract_tibbles`
+  * `bind_table` is now `bind_tibbles`
+    * All old functions are still supported and deliver a deprecation warning
```
Comment (skadauke) [inline cran-comments.md:14]: ~deliver~ throw

File: cran-comments.md [hunk 10]
```diff
@@ -1,27 +1,46 @@
...
+* The `read_redcap` output now includes additional helpful metadata
+* New `make_labelled()` function allowing for descriptive labelling application to variables via the [`labelled`](https://larmarange.github.io/labelled/) package
+* New arguments available for `read_redcap` capabilities:
+  * `forms`
+  * `export_survey_fields`
+
+## Test environments
 
+1. Local Ubuntu, R 4.2.0
```
Comment (skadauke) [inline cran-comments.md:23]: Why local Ubuntu? Should this say Local macOS?

File: cran-comments.md [hunk 11]
```diff
@@ -1,27 +1,46 @@
...
+
+- 0 ERRORs or WARNINGs on any builds
 - A NOTE is generated in R-hub Fedora Linux (R-devel, clang, gfortran), but could not be reproduced locally. 
 
 ```
 * checking HTML version of manual ... NOTE
 Skipping checking HTML validation: no command 'tidy' found
 ```
 
+  - Based on [this discussion](https://groups.google.com/g/r-sig-mac/c/7u_ivEj4zhM?pli=1) we could not determine a way to resolve this internal to our code.
```
Comment (skadauke) [inline cran-comments.md:42]: Would use the exact same comment that Will Beasley used here

File: cran-comments.md [hunk 12]
```diff
@@ -1,27 +1,46 @@
...
+
 ## Downstream Dependencies:
 
-No downstream packages are affected. The package that depends/imports/suggests REDCapR passes checks with `revdepcheck::revdep_check()`. Results: https://github.com/CHOP-CGTDataOps/REDCapTidieR/blob/main/revdep/cran.md
+No downstream packages are affected. The package that depends/imports/suggests REDCapTidieR passes checks with `revdepcheck::revdep_check()`. Results: https://github.com/CHOP-CGTInformatics/REDCapTidieR/blob/main/revdep/cran.md
```
Comment (skadauke) [inline cran-comments.md:46]: ?! Which package depends/imports/suggests REDCapTidieR?

**Review by skadauke — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 1 above)
Comment (rsh52) [reply NEWS.md:12]: Done!

File: NEWS.md (hunk 2 above)
Comment (rsh52) [reply NEWS.md:25]: Done!

File: revdep/README.md (hunk 5 above)
Comment (rsh52) [reply revdep/README.md:21]: Yea probably, when we go for the actual submission today I'll bump and rerun and everything in `refresh.R`

File: cran-comments.md (hunk 10 above)
Comment (rsh52) [reply cran-comments.md:23]: Correct, my bad

File: cran-comments.md (hunk 12 above)
Comment (rsh52) [reply cran-comments.md:46]: This was taken from Will's, but references `ReviewR`. I'll remove and change to

```
No downstream packages are affected. Results: https://github.com/CHOP-CGTInformatics/REDCapTidieR/blob/main/revdep/cran.md
```
